app/src/main/java/com/example/sabonit/Product.java

```diff
@@ -69,15 +73,8 @@ public void setDepartment(String department) {
         this.department = department;
     }
 
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
-        this.name = name;
-    }
 
-    public String getFullName() { return this.department + " " + this.name; }
+    public String getFullName() { return this.department + " " + this.smell; }
```

Comment:
this function needs to be documented, it's not a regular getter.
you supposed to explain what is the purpose of it

Reply:
please do it yourself in the future